server/test/run

```diff
@@ -4,9 +4,9 @@
 # platforms (well, mostly matches on all platforms).
 umask 0002
 
-# Don't allow an external _PBENCH_SERVER_CONFIG environment variable to affect the operation
-# of the unit tests since the unit tests themselves use getconf.py which looks
-# for that envinronment variable as input.
+# Don't allow an external _PBENCH_SERVER_CONFIG environment variable to affect
+# the operation of the unit tests since the unit tests themselves use pbench-
+# config which looks for that envinronment variable as input.
```

Comment:
s/pbench-\nconfig/pbench-server-config/ ?

Reply:
I'll fix in the next commit.